node-require/calculate.js

```diff
@@ -0,0 +1,17 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+const x = process.argv[2];
+const y = process.argv[4];
+const operator = process.argv[3];
+
+if (operator === 'plus') {
+  console.log('result: ', add.add(x, y));
```

Comment:
After making the above change to the exports statement you can then just directly call the function here:
```JAVASCRIPT
console.log('result: ', add(x, y));
```